Doter's milestone 1 delivery

ChainBridgeNetworkTeam opened the pull request:
# Milestone Delivery Checklist

- [ ] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [ ] The [invoice form :pencil:](https://forms.gle/vgQEZzvvFUjCyBiW6) has been filled out for this milestone. 
- [ ] This pull request is being made by the same account as the accepted application. 
- [ ] In case of acceptance, the payment will be transferred to the BTC/ETH payment address provided in the application. 
- [ ] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/w3f/General-Grants-Program/blob/master/grants/milestone-deliverables-guidelines.md).

alxs commented:
Thank you for the delivery @ChainBridgeNetworkTeam. We'll look into it as soon as possible.

Noc2 commented:
@ChainBridgeNetworkTeam I started to look into your project. You can find my evaluation notes [here](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/evaluations/doter_1_Noc2.md). Could you take a look at them and potentially fix a few things. I guess the biggest issue is that I’m currently unable to update the metadata via polkadot.js/apps. Let me know if you have any questions.

ChainBridgeNetworkTeam commented:
> @ChainBridgeNetworkTeam I started to look into your project. You can find my evaluation notes [here](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/evaluations/doter_1_Noc2.md). Could you take a look at them and potentially fix a few things. I guess the biggest issue is that I’m currently unable to update the metadata via polkadot.js/apps. Let me know if you have any questions.

Thank you for the reply! I will fix the issue you mentioned in the evaluation.

ChainBridgeNetworkTeam commented:
Hello David! @Noc2  We are so glad to tell you that we had fixed all the bugs you mentioned in the evaluation!  
1. now Doter can response the metadata update call from dapp and show popup.  
2. we have adjusted the communication mechanism so that Doter and polkadot.js extension can coexist in same dapp and perform well.  
3. the UI bug that tiny image was shown in Ubuntu has been fixed.  
4. password that contain uppercase and lowercase alphabets, numbers and symbols will be marked as “strong“.  
5. “transfer all” will subtract miner’s fee automatically to make sure the transfer succeed.  

We also made other minor adjustments to improve user experience.  

The latest version of extension is in the directory “plugin”, under the branch “v1.0.3”. We will launch the new version 1.0.3 of Doter at Google extension store in recent days.

Noc2 commented:
Thanks for the update. I’m happy to confirm that everything works now. Looks really good. To be honest, I wasn’t expecting you to fix all the issues. This milestone is a pass. I will forward your invoice internally and feel free to reach out to grantspr@web3.foundation to announce the project, see https://github.com/w3f/General-Grants-Program/blob/master/grants/announcement-guidelines.md.

fededubbi commented:
Good morning,

Do you want to be paid in USDT?
On the invoice the amount is USD 6650.

Many thanks,
Regards,

Federica

ChainBridgeNetworkTeam commented:
Hello!
It's ok for you to pay in USDT, and our address
is 0x9bb9af8f2fd4487c88f1352fdd32a3ec2b6d28b6
Thank you for your reply!

Sincerely
yours chainbridgenetworkteam

fededubbi ***@***.***> 于2021年6月17日周四 下午5:24写道：

> Good morning,
>
> Do you want to be paid in USDT?
> On the invoice the amount is USD 6650.
>
> Many thanks,
> Regards,
>
> Federica
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/w3f/Grant-Milestone-Delivery/pull/181#issuecomment-863081503>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATL2ROQIGNIAXPGK3GCISVDTTG5MBANCNFSM45TFESOQ>
> .
>
